code/transportation-development-services/tds.js

```diff
@@ -614,6 +627,251 @@ $(document).on('knack-view-render.view_744', function(event, view, record) {
+          ${dropdownMenuItem(recordId, "tia-case-log", "fa-briefcase", "Case Log")}\
+        </ul>\
+      </li>\
+    </ul>\
+  </div>`).appendTo("#view_926")
+})
+
+/* Memo Builder Page */
+$(document).on('knack-view-render.view_889', function(event, view, record) {
+  var recordId = view.scene.scene_id;
+
+  $(`<div class="details-dropdown-menu tabs">\
+    <ul id="tia-menu-list">\
+      <li class="tia-dropdown-menu kn-dropdown-menu kn-button">\
+        <a href="#tia-requests/tia-case-details/${recordId}/tia-case-management/${recordId}" data-kn-slug="#case-management">\
+          <span class="nav-dropdown-link">Case Management</span>\
+          <span class="kn-dropdown-icon fa fa-caret-down" />\
+        </a>\
+        <ul class="kn-dropdown-menu-list tia-dropdown-menu-list" style="min-width: 152px; margin: 0;">\
+          ${dropdownMenuItem(recordId, "tia-case-details", "fa-list-alt", "Case Details")}\
+          ${dropdownMenuItem(recordId, "tia-case-management", "fa-archive", "Scope & Submissions")}\
+          ${dropdownMenuItem(recordId, "tia-mitigation-details", "fa-file-text-o", "Mitigations")}\
+          ${dropdownMenuItem(recordId, "memo-builder", "fa-medium", "Memo Builder")}\
+          ${dropdownMenuItem(recordId, "tia-case-log", "fa-briefcase", "Case Log")}\
+        </ul>\
+      </li>\
+      ${dropdownMenuItem(recordId, "new-final-memo", "fa-plus-square", "New Final Memo")}\
+    </ul>\
+  </div>`).appendTo("#view_889")
+})
+
+/* Case Log Page */
+$(document).on('knack-view-render.view_893', function(event, view, record) {
+  var recordId = view.scene.scene_id;
+
+  $(`<div class="details-dropdown-menu tabs">\
+    <ul id="tia-menu-list">\
+      <li class="tia-dropdown-menu kn-dropdown-menu kn-button">\
+        <a href="#tia-requests/tia-case-details/${recordId}/tia-case-management/${recordId}" data-kn-slug="#case-management">\
+          <span class="nav-dropdown-link">Case Management</span>\
+          <span class="kn-dropdown-icon fa fa-caret-down" />\
+        </a>\
+        <ul class="kn-dropdown-menu-list tia-dropdown-menu-list" style="min-width: 152px; margin: 0;">\
+          ${dropdownMenuItem(recordId, "tia-case-details", "fa-list-alt", "Case Details")}\
+          ${dropdownMenuItem(recordId, "tia-case-management", "fa-archive", "Scope & Submissions")}\
+          ${dropdownMenuItem(recordId, "tia-mitigation-details", "fa-file-text-o", "Mitigations")}\
+          ${dropdownMenuItem(recordId, "memo-builder", "fa-medium", "Memo Builder")}\
+          ${dropdownMenuItem(recordId, "tia-case-log", "fa-briefcase", "Case Log")}\
+        </ul>\
+      </li>\
+    </ul>\
+  </div>`).appendTo("#view_893")
+})
+
+/* Scope Submission Details Page */
+$(document).on('knack-view-render.view_901', function(event, view, record) {
+  // regex: match the 24 digit record id that comes in hash part of url after "/"
+  const recordId = window.location.hash.match(/(?<=\/)\d[a-z0-9]{23}/)[0]
```

Comment:
here is the update to get the record Id from the url

Reply:
In testing it looks like the Case Management menu is navigating correctly with the new regex but the other menu buttons are now having trouble navigating.
For example:
![image](https://user-images.githubusercontent.com/67915167/123998996-a9621a80-d997-11eb-877d-8b867895cf42.png)

`${dropdownMenuItem(recordId, "assign-submission-reviewer", "fa-hand-o-up", "Assign Reviewer")}\
      ${dropdownMenuItem(recordId, "begin-submission-review", "fa-play-circle-o", "Begin Review")}\
      ${dropdownMenuItem(recordId, "complete-submission-review", "fa-exchange", "Reject/Approve")}\
      ${dropdownMenuItem(recordId, "submission-change-log", "fa-list-ol", "Submission Log")}\`

I believe its because they use the same variable `var recordId = view.scene.scene_id;`

Should we rename either the const or the var?